website/docs/r/cloudiot_registry.html.markdown

```diff
@@ -0,0 +1,116 @@
+---
+layout: "google"
+page_title: "Google: google_cloudiot_registry"
+sidebar_current: "docs-google-cloudiot-registry-x"
+description: |-
+  Creates a device registry in Google's Cloud IoT Core platform
+---
+
+# google\cloudiot\_registry
+
+ Creates a device registry in Google's Cloud IoT Core platform. For more information see
+[the official documentation](https://cloud.google.com/iot/docs/) and
+[API](https://cloud.google.com/iot/docs/reference/rest/v1/projects.locations.registries).
+
+
+## Example Usage
+
+```hcl
+resource "google_pubsub_topic" "default-devicestatus" {
+  name = "default-devicestatus"
+}
+
+resource "google_pubsub_topic" "default-telemetry" {
+  name = "default-telemetry"
+}
+
+resource "google_cloudiot_registry" "default-registry" {
+  name = "default-registry"
+
+  event_notification_configs = [{
+    pubsub_topic_name = "${google_pubsub_topic.default-devicestatus.id}"
+  }]
+
+  state_notification_config = {
+    pubsub_topic_name = "${google_pubsub_topic.default-telemetry.id}"
+  }
+
+  http_config = {
+    http_enabled_state = "HTTP_ENABLED"
+  }
+
+  mqtt_config = {
+    mqtt_enabled_state = "MQTT_ENABLED"
+  }
+
+  credentials = [
+    {
+      public_key_certificate = {
+        format      = "X509_CERTIFICATE_PEM"
+        certificate = "<CERTIFICATE>"
```

Comment:
Wouldn't hurt to do, maybe, "${file("foo.pem")}" or something like that so the example is more valid?

Reply:
Yep! Sounds much better!